README.md

```diff
@@ -23,6 +23,7 @@ The following entry points are provided:
 - rundnsmasq - Runs the dnmasq dhcp server to provide addresses and initiate PXE boot of baremetal nodes.  This includes a lightweight TFTP server.  Details on dnsmasq can be found at http://www.thekelleys.org.uk/dnsmasq/doc.html.
 - runhttpd - Starts the Apache web server to provide images via http for PXE boot and for deployment of the final images.
 - runmariadb - Provides a database to store information associated with baremetal nodes.
+- ironic-provisioning-log-watch - Waits for host provisioning ramdisk logs to appear, prints their contents and deletes files.
```

Comment:
Maybe name it consistently, like `runlogwatch`?

Reply:
Done.